### PR TITLE
Hotfix prompt and model

### PR DESCRIPTION
**Pull request by ryukinix**

### CODE REVIEW

File: nemli/commands/utility/summarize.py [hunk 1]
```diff
@@ -47,15 +47,15 @@ async def summarize_command(
         # Here we call the OpenAI API to create a summary of the last 100
         # messages in the channel with the previously created variables
         response = openai_client.chat.completions.create(
-            model="gpt-3.5-turbo-16k",
+            model="gpt-4o",
```
Comment (rdenadai) [inline nemli/commands/utility/summarize.py:50]: troca para: `gpt-4o-mini`

**Review by rdenadai — CHANGES_REQUESTED** [review]
modelo apenas

**Review by mattpsvreis — DISMISSED** [review]
LGTM!

File: nemli/commands/utility/summarize.py (hunk 1 above)
Comment (mattpsvreis) [reply nemli/commands/utility/summarize.py:50]: ## GPT-4o
### Pros:

- **Optimized for Performance:** Better fine-tuned for specific tasks, potentially offering faster and more accurate responses.
Improved Context Handling: Enhanced capability to manage large context windows efficiently.
- **Cost-Effective:** Optimized versions often offer a balance between performance and cost.

### Cons:

**Specialization Limitations:** May not be as versatile across different tasks as the standard GPT-4 model.

## GPT-4o Mini
### Pros:

- **Efficiency:** Designed to be more resource-efficient, reducing operational costs.
- **Faster Response Times:** Lower latency due to reduced model size.
- **Cost-Effective:** More affordable, suitable for frequent use cases.

### Cons:

- **Context Window:** Smaller context window compared to GPT-4, potentially limiting the ability to summarize very long conversations.
- **Performance:** Slightly reduced performance in complex language understanding tasks.

**Comment by mattpsvreis** [comment]
Covered by PR https://github.com/milharal-dev/nemli-nemlerei-bot/pull/14

**Pull request closed without merging**